data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1450,21 +1480,31 @@ descriptions:new_tribe {
                pgettext("atlanteans_building", "The fisher’s house needs water full of fish within the work area."),
                -- TRANSLATORS: Note helptext for an atlantean production site: Fisher's House, part 2
                pgettext("atlanteans_building", "Build a fish breeder’s house close to the fisher’s house to make sure that you don’t run out of fish.")
-            }
+            },
+            -- TRANSLATORS: Note lore for an atlantean production site: Fisher's House use some local fise song as you like
+            lore = pgettext("atlanteans_building", [[‘Take your net and come to the sea<br>]] ..
+                                          [[In the early morning the moon ya still see<br>]] ..
+                                          [[Collect ya catch of fish for us and for you<br>]] ..
+                 -- NOCOM / Better text needed: I don't understand the last line
+                                          [[And Satul will warm the people of from your do.']]),
```

Comment:
See annotation ;)

Reply:
collect ya catch by net or by spear, and don't forget to give Satul his share.